src/components/10-atoms/checkbox/index.js

```diff
@@ -168,6 +177,18 @@ class AXACheckbox extends NoShadowDOM {
       this.querySelector('input').checked = true;
       this.state.native = true;
     }
+
+    const iconElement = this.querySelector('.a-checkbox__icon');
```

Comment:
@NoahLiechti please use JS selectors fo JS

Reply:
@NoahLiechti i srtill can see `const iconElement = this.querySelector('.a-checkbox__icon');` instead of `const iconElement = this.querySelector('.js-checkbox__icon');`